packages/expo-notifications/src/TokenEmitter.ts

```diff
@@ -0,0 +1,24 @@
+import { EventEmitter, Subscription, Platform } from '@unimodules/core';
+import { DevicePushToken } from './getDevicePushTokenAsync';
+import PushTokenManager from './PushTokenManager';
+
+export type TokenListener = (token: DevicePushToken) => void;
```

Comment:
TokenListener -> PushTokenListener

(there are other types of tokens in the rest of the codebase or application)